tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_3d.py

```diff
@@ -0,0 +1,252 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from ..adapters import Adapter
+from ..adapters.pose_estimation import HumanPoseAdapter
+from ..config import ConfigValidator, StringField
+from ..representation import PoseEstimationPrediction, PoseEstimation3dPrediction
+
+AVG_PERSON_HEIGHT = 180
+
+# pelvis (body center) is missing, id == 2
+map_id_to_panoptic = [1, 0, 9, 10, 11, 3, 4, 5, 12, 13, 14, 6, 7, 8, 15, 16, 17, 18]
+
+limbs = [[18, 17, 1],
+         [16, 15, 1],
+         [5, 4, 3],
+         [8, 7, 6],
+         [11, 10, 9],
+         [14, 13, 12]]
+
+
+class HumanPose3dAdapter(Adapter):
+    __provider__ = 'human_pose_estimation_3d'
+    prediction_types = (PoseEstimation3dPrediction, )
+
+    def __init__(self, launcher_config, label_map=None, output_blob=None):
+        super().__init__(launcher_config, label_map, output_blob)
+        self.pose_adapter = HumanPoseAdapter(launcher_config, label_map, output_blob)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'features_3d_out': StringField(description="Name of output layer with 3d features."),
+            'keypoints_heatmap_out': StringField(description="Name of output layer with keypoints heatmaps."),
+            'part_affinity_fields_out': StringField(
+                description="Name of output layer with keypoints pairwise relations (part affinity fields)."
+            )
+        })
+
+        return parameters
+
+    def validate_config(self):
+        super().validate_config(on_extra_argument=ConfigValidator.WARN_ON_EXTRA_ARGUMENT)
+
+    def configure(self):
+        self.features_3d = self.get_value_from_config('features_3d_out')
+        self.part_affinity_fields = self.get_value_from_config('part_affinity_fields_out')
+        self.keypoints_heatmap = self.get_value_from_config('keypoints_heatmap_out')
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        result = []
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+        raw_output = zip(
+            identifiers, raw_outputs[self.features_3d], raw_outputs[self.keypoints_heatmap],
+            raw_outputs[self.part_affinity_fields], frame_meta
+        )
+        for identifier, features, heatmap, paf, meta in raw_output:
+            poses_2d = self._extract_poses_2d(heatmap, paf, meta, identifier)
+            height, width, _ = meta['image_size']
+            scale_y = height / features.shape[1]
+            scale_x = width / features.shape[2]
+            panoptic_poses_3d, translations, panoptic_poses_2d = HumanPose3dAdapter._parse_poses(
+                features, poses_2d, scale_y, scale_x, 1 / scale_x)
+                                                      #np.float32(meta['fx']))
```

Comment:
@eaidova Don't know how to pass value from annotations (`fx`) here.

Reply:
Hmm..Is `fx` stored in annotation?
Can it be made in postprocessor, not adapter? (Postprocessors have access for both annotation and prediction)
Or moved to image metadata?